CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## Release (2024-XX-XX)
+- `iaas`: [v0.8.0](services/iaas/CHANGELOG.md#v080-2024-08-21)
+  - **Feature:** `CreateNetworkIPv4Body` and `CreateNetworkIPv6Body` have a new field `Prefix`
```

Comment:
Do you know if the new field is required? If yes this will be a breaking change, as `CreateNetworkIPv6Body` is used in the `CreateNetworkPayload`

Reply:
Prefix is not a required field. It is not marked in API docs as required and I verifies via Postman